### PR TITLE
Explicitly state the accuracy for ldexp

### PR DESCRIPTION
**Pull request by zoddicus**
Issue #2631

### CODE REVIEW

**Comment by zoddicus** [comment]
PTAL, just want to be explicit about this, since as written it isn't obvious this is an inherited accuracy.

File: wgsl/index.bs [hunk 1]
```diff
@@ -8686,7 +8686,7 @@ value with the same sign.
   <tr><td>`fract(x)`<td colspan=2 style="text-align:left;">Inherited from `x - floor(x)`
   <tr><td>`frexp(x)`<td colspan=2 style="text-align:left;">Correctly rounded
   <tr><td>`inverseSqrt(x)`<td colspan=2 style="text-align:left;">2 ULP
-  <tr><td>`ldexp(x, y)`<td colspan=2 style="text-align:left;">Correctly rounded
+  <tr><td>`ldexp(x, y)`<td colspan=2 style="text-align:left;">Inherited from `x * pow(2, y)`
```
Comment (dneto0) [inline wgsl/index.bs:8689]: I don't think so?

ldexp can (and should?!) be implemented by directly manipulating the exponent bits of the number.

**Review by dneto0 — CHANGES_REQUESTED** [review]

File: wgsl/index.bs (hunk 1 above)
Comment (zoddicus) [reply wgsl/index.bs:8689]: I was just basing this off of what the spec says ldexp does, from https://www.w3.org/TR/WGSL/#float-builtin-functions
```
@const fn
ldexp(e1: T ,e2: I ) -> T	Returns e1 * 2e2. Component-wise when T is a vector.
```

The text in the Vulkan spec goes into more detail about the edge cases and implementation details for ldexp. We may need to expand/clarify ldexp.

Comment (zoddicus) [reply wgsl/index.bs:8689]: I don't think I had groked the fact that ldexp is literally meant to build a floating point number on a bit level. Re-reading your comments and the defintion in SPIR-V it finally sunk in.

I will close this out.

**Pull request closed without merging**